frontend/theta-project-tracker/src/app/components/new-user.component.ts

```diff
@@ -100,60 +101,85 @@ import { EventEmitter } from '@angular/core';
     .cost {
       max-width: 100px
     }
+    mat-icon:hover {
+      cursor: pointer;
+    }
     `
   ]
 })
 export class NewUserComponent implements OnInit {
 
   constructor(private userService: UserService) { }
-  newUser: FormGroup;
+
+  newUser = new FormGroup({
+    firstName: new FormControl(null, Validators.required),
+    lastName: new FormControl(null, Validators.required),
```

Comment:
firstnam lastname-hez mehetne olyan validátor ami figyeli hogy ne lehessen csupa whitespacet beírni